module/actor/actor.js

```diff
@@ -1796,35 +1796,13 @@ export class HeroSystem6eActor extends Actor {
 
         return changed;
     }
-
+   
     updateRollable(key) {
         const characteristic = this.system.characteristics[key];
         if (characteristic.type === "rollable") {
-            if (characteristic.value === 0) {
-                characteristic.roll = 8;
-            } else if (characteristic.value <= 2) {
-                characteristic.roll = 9;
-            } else if (characteristic.value <= 7) {
-                characteristic.roll = 10;
-            } else if (characteristic.value <= 12) {
-                characteristic.roll = 11;
-            } else if (characteristic.value <= 17) {
-                characteristic.roll = 12;
-            } else if (characteristic.value <= 22) {
-                characteristic.roll = 13;
-            } else if (characteristic.value <= 27) {
-                characteristic.roll = 14;
-            } else if (characteristic.value <= 32) {
-                characteristic.roll = 15;
-            } else if (characteristic.value <= 37) {
-                characteristic.roll = 16;
-            } else if (characteristic.value <= 42) {
-                characteristic.roll = 17;
-            } else if (characteristic.value <= 47) {
-                characteristic.roll = 18;
-            } else if (characteristic.value <= 52) {
-                characteristic.roll = 19;
-            } else {
+            if (characteristic.value >= 0 && characteristic.value <= 52) {
```

Comment:
is there a cap on negative values for 5e? If not that wouldn't make much sense past -47 since anything after that would still giving you a negative roll modifier, but yeah thank you for bringing that up. I personally don't have a 5e rulebook on hand. This should be an easy change

Reply:
Arguably anything below 3 is a problem, but technically 3 is always a success.

Reply:
Oh, but a GM could decide that there's some kind of a modifier to apply to roll so that can always bring it back up from less than 3.